jsFiles/utility.js

```diff
@@ -63,17 +63,39 @@ function alertError(error) {
  *
  * @param {number} timeStamp - unix, contains information about the time.
  */
-// TODO: fix for when displaying the time of a searched city -- timezones offset
-function timeConversion(timeStamp) {
-  // The multiplication *1000 is because the timeStamp is in second
-  // and the Date expects miliseconds
-  const timeObj = new Date(timeStamp * 1000);
-  return timeObj.toLocaleTimeString("en-US", {
+function timeConversion(timeStamp, offsetTime) {
```

Comment:
hm...this function looks similar to the `cityTime`. Maybe you need only one of them, since the difference is only in what you return?

Reply:
I saw this before but I couldn't come up with something to place them together before the pull request, now I just got an idea :)